### ISSUE TITLE
Regarding the training configuration for the Post-train experiment on SimplerEnv-WidowX+Bridge

### ISSUE THREAD

**Issue by Oliverbansk** [Robot-VLAs/RoboVLMs#27]
Hi! Great work!
I'm writing to inquire the training configuration for the Post-train experiment on SimplerEnv-WidowX+Bridge. 

I notice that `configs/oxe_training/finetune_kosmos_cont-lstm-post_full-ft_text_vision_wd-0_use-hand_ws-16_act-10_bridge_finetune.json` already contained the training configuration for the Bridge Finetune experiment, which finetune the base VLM model with the entire Bridge-V2 dataset, but the Post-train experiment's configuration is not included (i.e. First training the VLMs on OXE, followed by fine-tuning with in-doman Bridge-V2  data). 

Simply modifying the `configs/oxe_training/finetune_kosmos_cont-lstm-post_full-ft_text_vision_wd-0_use-hand_ws-16_act-10_bridge_finetune.json` file doesn't yield accurate results for Post-train. Maybe the training timesteps is different?

Could you please share the Post-train experiment's configuration? Thank you so much!

**Comment by LukeLIN-web** [comment]
I have a similar question  https://github.com/Robot-VLAs/RoboVLMs/issues/29#issue-2994772784 

What did you means about doesn't yield accurate results for Post-train? Is it very bad 0% accuracy?

**Comment by Oliverbansk** [comment]
The Put Spoon task has a 40% accuracy in my Post-train run while other tasks have 0% accuracy. I'm thinking about maybe the training timestep should be more than 50000?

**Comment by LukeLIN-web** [comment]
> The Put Spoon task has a 40% accuracy in my Post-train run while other tasks have 0% accuracy. I'm thinking about maybe the training timestep should be more than 50000?

Yeah, I saw spatialvla says requires 1-3 epoch.   fractual  3.7M,  78883 steps one epoch,   batch size 48 need 78k-180k steps.

**Comment by XXXUAN0904** [comment]
> 我有一个类似的问题 [#29 （评论）](https://github.com/Robot-VLAs/RoboVLMs/issues/29#issue-2994772784)
> 
> 你说的什么没有为 Post-train 产生准确的结果？是不是 0% 的准确率很差？

How about training for more than 50,000 steps? I'm training for 10,000 steps and all my success rates are 0.

**Comment by XXXUAN0904** [comment]
> I have a similar question [#29 (comment)](https://github.com/Robot-VLAs/RoboVLMs/issues/29#issue-2994772784)
> 
> What did you means about doesn't yield accurate results for Post-train? Is it very bad 0% accuracy?

How about training for more than 50,000 steps? I'm training for 10,000 steps and all my success rates are 0.

**Comment by Oliverbansk** [comment]
In my experiment (recreating Post-Train), fine-tuning for 400000 timesteps yielded comparable results with the paper's.

**Comment by LukeLIN-web** [comment]
> In my experiment (recreating Post-Train), fine-tuning for 400000 timesteps yielded comparable results with the paper's.

I am using 20 hour 5 GPU, achieve  40k steps, bs 8. 
Crazy...  It seems need 200 hours for me . 
Thank you for your effort and sharing!

**Comment by XXXUAN0904** [comment]
> Hi! Great work! I'm writing to inquire the training configuration for the Post-train experiment on SimplerEnv-WidowX+Bridge.
> 
> I notice that `configs/oxe_training/finetune_kosmos_cont-lstm-post_full-ft_text_vision_wd-0_use-hand_ws-16_act-10_bridge_finetune.json` already contained the training configuration for the Bridge Finetune experiment, which finetune the base VLM model with the entire Bridge-V2 dataset, but the Post-train experiment's configuration is not included (i.e. First training the VLMs on OXE, followed by fine-tuning with in-doman Bridge-V2 data).
> 
> Simply modifying the `configs/oxe_training/finetune_kosmos_cont-lstm-post_full-ft_text_vision_wd-0_use-hand_ws-16_act-10_bridge_finetune.json` file doesn't yield accurate results for Post-train. Maybe the training timesteps is different?
> 
> Could you please share the Post-train experiment's configuration? Thank you so much!

Hi!Did you solve the problem, the code doesn't seem to provide runs/checkpoints/oxe_pretrain/kosmos/2024-11-21/01-16/epoch=0-step=50000.pt this file, how did you do the post-train?
